core/build.gradle.kts

```diff
@@ -61,6 +61,8 @@ dependencies {
     implementation("commons-io:commons-io")
     implementation("org.codehaus.janino:commons-compiler")
     implementation("org.codehaus.janino:janino")
+    annotationProcessor("org.immutables:value:2.8.2")
```

Comment:
Although, website still mentions 2.8.2, the latest release according to Maven Central and Github is 2.8.8 (with 2.9.0 being rc'ed)

Reply:
fixed